pipeline/src/lib.rs

```diff
@@ -46,7 +49,7 @@ mod tests {
   id: stream_rtsp_out1
   properties:
     uri: rtsp://127.0.0.1:5555/mycamera
-    udp_port: '5800'
+    udp_port: 5800
```

Comment:
Please note that this property is being inserted by `lumeod` and therefore we control its type, so it can be just a regular `u16`, not a `String`.

Reply:
Cool, less manual Deserialize already. We need to update the testcase in lumeo-gst though?

Reply:
Yeah, and not only testcase :)

Reply:
only realized now that you don't need the `#[serde(serialize_with = "super::serialize_required")]` for this field now.

Reply:
ah nm, you've removed that too. Got confused during merge conflict resolution. :)